### DISCUSSION TITLE
How to Auto generate Migrations based on Model update ?

### DISCUSSION THREAD

**Discussion by VaZark** [sequelize/sequelize#14449] [Help & Questions]
Are there any ways to introspect the current state of the db, compare it with the model definition and generate migrations based on their diff?

**Comment by ephys** [comment]
You can try https://github.com/MRVMV/sequelize-mig. As far as I know, it's the only library that attempts to do this for sequelize https://sequelize.org/docs/v7/other-topics/resources/#migrations
